views/login/notice.pug

```diff
@@ -0,0 +1,20 @@
+extends ../base
+
+block variables
+  -var title = __('Notice of assessment')
+  -var noticeOfAssessment = !hasData(data, 'login.noticeOfAssessment') ? '' : data.login.noticeOfAssessment ? 'Yes' : 'No'
+
+block content
+
+  h1 #{title}
+
+  p #{__('CRA sends your notice of assessment about 2 weeks after you file taxes.')}
+  p #{__('You can download the notice immediately after filing by answering security questions. You will be able to print or save the downloaded notice.')}
+  p #{__('The questions ask for information such as your previous addresses and banking details. You may need to check your records to find the answers.')}
```

Comment:
We should tell them that they don't need to answer the security questions to file their taxes, as it's not explicit.